### PR TITLE
Add SaveOptions description to tutorial

### PR DESCRIPTION
**Pull request by k-w-w**
Adding `tf.keras.util.SaveOptions` segment to the tutorial, along with a few other minor rewordings.

### CODE REVIEW

**Review by fchollet — COMMENTED** [review]
Thanks for the PR. This file is autogenerated, please only update the source of truth, located at `guides/serialization_and_saving.py`.

**Review by fchollet — COMMENTED** [review]
This PR contains many files that it doesn't actually modify. Please rebase.

File: guides/serialization_and_saving.py [hunk 1]
```diff
@@ -177,26 +181,52 @@ def call(self, inputs):
...
+model is loaded by dynamically creating the model class that acts like the original model.
+"""
+
 
 """
-As seen in the example above, the loader dynamically creates a new model class
-that acts like the original model.
+#### Configuring the SavedModel
+
+*New in TensoFlow 2.4*
```
Comment (fchollet) [inline guides/serialization_and_saving.py:225]: Please update this to reflect the final API

Comment (k-w-w) [reply guides/serialization_and_saving.py:225]: Done

**Review by fchollet — APPROVED** [review]
LGTM, thank you

**Pull request merged**